template.adoc

```diff
@@ -63,6 +63,8 @@ include::{common-includes}/trywhatyoubuild-intro.adoc[]
 
 include::{common-includes}/trywhatyoubuild-end.adoc[]
 
+//Suggest readers to navigate to the `start` directory in the section where user starts working with the implementation.

```

Comment:
Suggest an example, to make it a more standardized statement to help keep things consistent.